app/src/common/constants/projectRoles.js

```diff
@@ -20,6 +20,11 @@ export const CUSTOMER = 'CUSTOMER';
 export const OPERATOR = 'OPERATOR';
```

Comment:
Can we remove deprecated roles?

Reply:
The old roles are used in components that are out of scope. They will be deleted later.